gtk/Utils.cc

```diff
@@ -154,14 +152,14 @@ void gtr_add_torrent_error_dialog(Gtk::Widget& child, tr_torrent* duplicate_torr
 
     if (duplicate_torrent != nullptr)
     {
-        secondary = gtr_sprintf(
-            _("The torrent file \"%s\" is already in use by \"%s.\""),
-            filename,
-            tr_torrentName(duplicate_torrent));
+        secondary = fmt::format(
+            _("The torrent file '{path}' is already in use by '{torrent_name}'."),
+            fmt::arg("path", filename),
+            fmt::arg("torrent_name", tr_torrentName(duplicate_torrent)));
     }
     else
     {
-        secondary = gtr_sprintf(_("Unable to add torrent file \"%s\"."), filename);
+        secondary = fmt::format(_("Couldn't add .torrent file '{path}'"), fmt::arg("path", filename));
```

Comment:
Not sure how I feel about this "torrent" -> ".torrent" move. Both ["torrent file"](https://en.wikipedia.org/wiki/Torrent_file) and ["magnet link"](https://en.wikipedia.org/wiki/Magnet_URI_scheme) seem to be accepted domain-specific terms. Currently in `main` we have 39 instances of ".torrent file" and 57 instances of "torrent file".

Same in Session.cc.

Reply:
90074d7dbc5cbaa59ede0bb74cb3b085c4bc0ee9